flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitor.java

```diff
@@ -435,6 +424,36 @@ protected void initChannel(SocketChannel ch) {
 		LOG.info("Web frontend listening at " + address + ':' + port);
 	}
 
+	private void GET(Router router, RequestHandler handler) {
```

Comment:
Can we add a high level comment that says that we have these helpers in order to do the route path setup?